public/css/main.css

```diff
@@ -434,17 +436,136 @@ video#bgvid {
     color: #2D2A3F;
 
 }
-#difference-container{
-    position:relative;
-    width: 100%;
+
+.ps-skyline {
+    position: relative;
+    padding: 15px;
+}
+
+.ps-skyline-overlay {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    display: flex;
+    flex-direction: column;
+}
+
+.ps-skyline-overlay > * {
+    align-self: center;
+    margin-top: 20px;
+}
+
+#skyline-spacing {
+    margin: 0 auto; 
+    width: fit-content; 
+    margin-top: 135px;
+}
+
+.ps-skyline-overlay > .row {
+    background-color: white;
+    opacity: 90%;
+    padding: 5px;
+    border-radius: 15px;
+    max-width: 100%;
+    font-family: raleway, sans-serif;
+    letter-spacing: 0em;
+    font-size: 45px;
+    font-style: normal;
+    font-variant: normal;
+    line-height: 1.1em;
+    color: #2D2A3F;
+} 
+
+.ps-skyline-overlay-btn {
+    background-color: #eb734d;
+    opacity: 90%;
+    -moz-border-radius:15px;
+    -webkit-border-radius:15px;
+    border-radius:15px;
+    border:1px solid #2D2A3F;
+    cursor:pointer;
+    color:#2D2A3F;
+    font: normal normal normal 14px/1.4em raleway,sans-serif;
+    font-family: 'Raleway', sans-serif;
+    font-size:20px;
+    font-weight: bold;
+    padding:10px 45px;
+    text-decoration:none;
+    transition: border-color 0.4s ease 0s, background-color 0.4s ease 0s;
+}
+
+.ps-skyline-overlay-btn:hover {
+    border:1px solid #eb734d;
+    color: #ffffff;
+    text-shadow: none;
+}
+
+.ps-skyline-container {
+    display: flex;
+    flex-direction: column;
+}
+
+.ps-skyline-container > img {
+    width: 1400px;
+    align-self: center;
 }
 
-.grayButton{
+.skyline-cloud-overlay {
+    margin-top: -68px; /* Height of Cloud .png */
+}
+
+/* TODO: Change from ID to class, remove unneeded css */
```

Comment:
Is this still an open TODO?

Reply:
Ah, no that's closed. Not relevant anymore, removing comment now.